recipes/LibriSpeech/SSL/train.py

```diff
@@ -0,0 +1,398 @@
+#!/usr/bin/env python3
+"""Recipe for pretraining wav2vec2. See config file for model definition.
+
+To run this recipe call python train.py hparams/train_it_wav2vec.yaml --find_unused_parameters --max_grad_norm 0.0
+"""
+
+import os
+import os.path as osp
+import re
+from tqdm.contrib import tqdm
+import logging
+import sys
+import time
+import tempfile
+import pathlib
+import functools
+from contextlib import ExitStack
+
+from types import SimpleNamespace
+import speechbrain as sb
+import torchaudio
+import torch
+import torch.nn.functional as F
+from torch.nn.parallel import DistributedDataParallel
+from torch.utils.data import DataLoader
+from hyperpyyaml import load_hyperpyyaml
+
+from speechbrain import Stage
+from speechbrain.utils.distributed import run_on_main
+from speechbrain.dataio.dataloader import SaveableDataLoader, LoopedLoader
+from speechbrain.dataio.sampler import DynamicBatchSampler, DistributedSamplerWrapper
+from speechbrain.lobes.models.wav2vec import make_padding_mask, compute_mask, W2VPaddedBatch
+
+
+PYTHON_VERSION_MAJOR = 3
+PYTHON_VERSION_MINOR = 7
+
+
+logger = logging.getLogger(__name__)
+
+
+@sb.utils.checkpoints.register_checkpoint_hooks
```

Comment:
Why is this needed ?

Reply:
I was overwriting _save and _recover before, think it can be removed now, will do